apiserver/facades/client/applicationoffers/applicationoffers_test.go

```diff
@@ -1269,13 +1269,56 @@ func (s *consumeSuite) TestRemoteApplicationInfo(c *gc.C) {
 	})
 }
 
-func (s *consumeSuite) TestDestroyOffers(c *gc.C) {
+func (s *consumeSuite) TestDestroyOffersNoForce(c *gc.C) {
 	s.setupOffer()
 	st := s.mockStatePool.st[testing.ModelTag.Id()]
 	st.(*mockState).users["foobar"] = &mockUser{"foobar"}
+	st.(*mockState).connections = []applicationoffers.OfferConnection{
+		&mockOfferConnection{
+			username:    "fred",
+			modelUUID:   testing.ModelTag.Id(),
+			relationKey: "hosted-db2:db wordpress:db",
+			relationId:  1,
+		},
+	}
+
+	s.authorizer.Tag = names.NewUserTag("admin")
+	results, err := s.api.DestroyOffers(params.DestroyApplicationOffers{
+		OfferURLs: []string{
+			"fred/prod.hosted-mysql"},
+	})
+	c.Assert(err, jc.ErrorIsNil)
+	c.Assert(results.Results, gc.HasLen, 1)
+	c.Assert(results.Results, jc.DeepEquals, []params.ErrorResult{
+		{
+			Error: &params.Error{Message: `offer has 1 relations`},
```

Comment:
'1 relations' sounds funny... I thought we had a common utility that would pluralize given nouns :D